### PR TITLE
[#ICC-150]  Add feature flag FF_PAYMENT_STATUS_ENABLED

### PR DESCRIPTION
**Pull request by gquadrati**
#### List of Changes

- Add FF_PAYMENT_STATUS_ENABLED, with false as default

#### Motivation and Context

This is needed for publishing #212 disabled, waiting the Payment Updater to be up and running

#### How Has This Been Tested?

Unit test

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: GetMessage/handler.ts [hunk 1]
```diff
@@ -435,6 +450,7 @@ export const GetMessageHandler = (
  */
 // eslint-disable-next-line max-params, prefer-arrow/prefer-arrow-functions
 export function GetMessage(
+  { FF_PAYMENT_STATUS_ENABLED }: IConfig,
```
Comment (AleDore) [inline GetMessage/handler.ts:453]: Why destructuring object here instead of passing a simple boolean?

File: utils/config.ts [hunk 2]
```diff
@@ -75,6 +77,8 @@ export const IConfig = t.intersection([
     FF_INCOMPLETE_SERVICE_WHITELIST: CommaSeparatedListOf(ServiceId),
     FF_OPT_IN_EMAIL_ENABLED: t.boolean,
 
+    FF_PAYMENT_STATUS_ENABLED: withFallback(BooleanFromString, false),
```
Comment (AleDore) [inline utils/config.ts:80]: <3

File: GetMessage/handler.ts (hunk 1 above)
Comment (gquadrati) [reply GetMessage/handler.ts:453]: Just in case we'll need more values in future, nothing else actually.. What do you prefer?

Comment (AleDore) [reply GetMessage/handler.ts:453]: Since we have to remove it once we have a stable backend service, It's ok :D

**Review by AleDore — APPROVED** [review]

File: GetMessage/handler.ts (hunk 1 above)
Comment (gquadrati) [reply GetMessage/handler.ts:453]: Right :D

**Pull request merged**